main.css

```diff
@@ -2,29 +2,30 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
- }
+}
 
```

Comment:
taki komentarz do całości. Odpaliłem sobie ten kod i takie propozycje z mojej strony (do których oczywiście nie musisz się zastosować):
- dla tytułu `Newsletter` na niższych rozdzielczościach `text-align: center` bo dziwnie to wygląda jak na mobilkach masz wszystko wyśrodkowane a ten tytuł do lewej 
- co do inputa troche mi brakuje kontrastu gdy robisz focus na nim. Może zrób tak żeby na focusie czcionka zmieniała kolor na biały natomiast tło było trochę ciemniejsze - np. takie: `#47b8b4BF` lub `#47b8b4B2`. Poza tym co myślisz o zmniejszeniu czasu tranzycji z 1s do .5s ?
- co do buttona `zapisz się` to usunąłbym na focusie ten outline bo prostokątne podświetlenie troche kłóci się z tą animacją buttona. wystarczy że w stylach dla buttona w focus dasz `outline: 0` - to powinno zadziałać ;) 

oczywiście zrobisz jak uważasz bo to tylko stylistyczne rzeczy. Takie drobne sugestie ;)